.github/workflows/add-ubuntu-latest-apt-mirrors.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+# run as root
+
+touch /etc/apt/sources.list.d/ubuntu-latest-mirrors.list
+for mirror in \
+    'http://ubuntu.osuosl.org/ubuntu/' \
+    'http://mirrors.ocf.berkeley.edu/ubuntu/' \
```

Comment:
I picked this one at random: given it's from a university I don't expect it will go away any time soon.
https://launchpad.net/ubuntu/+mirror/mirrors.ocf.berkeley.edu-archive